### PR TITLE
[Lighthouse plugin] Change setup wording of intro card to latest

### PR DESCRIPTION
**Pull request by tadashi0713**
## Hey, I just made a Pull Request!

Setup guide in intro card is outdated, so I updated as same as [README part](https://github.com/backstage/backstage/tree/master/plugins/lighthouse#getting-started)

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4738-dix5zfptxhu2lhgaxdnhcvqnqdyxsjct.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603cc74af285af07df9b14a5

File: plugins/lighthouse/src/components/Intro/index.tsx [hunk 1]
```diff
@@ -41,31 +41,37 @@ To get started, you will need a running instance of [lighthouse-audit-service](h
...
-} from '@backstage/plugin-lighthouse';
-
-const builder = ApiRegistry.builder();
+// At the top imports
+import { Router as LighthouseRouter } from '@backstage/plugin-lighthouse';
+
+// Inside App component
+<Routes>
+  // ...
+  <Route path="/lighthouse/*" element={<LighthouseRouter />} />
```
Comment (Rugvip) [inline plugins/lighthouse/src/components/Intro/index.tsx:65]: This is actually the following now 😅 

```tsx
    <Route path="/lighthouse" element={<LighthousePage />} />
```

File: plugins/lighthouse/src/components/Intro/index.tsx [hunk 2]
```diff
@@ -41,31 +41,37 @@ To get started, you will need a running instance of [lighthouse-audit-service](h
...
-  lighthouseApiRef,
-  LighthouseRestApi,
-} from '@backstage/plugin-lighthouse';
-
-const builder = ApiRegistry.builder();
+// At the top imports
+import { Router as LighthouseRouter } from '@backstage/plugin-lighthouse';
+
+// Inside App component
+<Routes>
```
Comment (Rugvip) [inline plugins/lighthouse/src/components/Intro/index.tsx:63]: This is now `<FlatRoutes>`

File: plugins/lighthouse/src/components/Intro/index.tsx [hunk 3]
```diff
@@ -41,31 +41,37 @@ To get started, you will need a running instance of [lighthouse-audit-service](h
...
 
 \`\`\`js
 export { plugin as LighthousePlugin } from '@backstage/plugin-lighthouse';
 \`\`\`
 
-Then, you need to use the \`lighthouseApiRef\` exported from the plugin to initialize the Rest API in
-your [\`apis.ts\`](https://github.com/backstage/backstage/blob/master/packages/app/src/apis.ts).
+Modify your app routes in \`App.tsx\` to include the Router component exported from the plugin, for example:
 
 \`\`\`js
```
Comment (Rugvip) [inline plugins/lighthouse/src/components/Intro/index.tsx:58]: Would assume that this looks better with `jsx` or `tsx`, but not sure 😅

**Review by Rugvip — COMMENTED** [review]
Thanks! 👍 

Just need it to be up to date with the even more latest version of things :grin:

**Comment by tadashi0713** [comment]
Thanks @Rugvip, I updated

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4738-dix5zfptxhu2lhgaxdnhcvqnqdyxsjct.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603cc74af285af07df9b14a5

**Review by Rugvip — APPROVED** [review]
Thanks, looks good! :shipit:

**Pull request merged**